src/main/java/seedu/duke/Parser.java

```diff
@@ -70,12 +70,30 @@ public static Command checkAction(String inputCommand) throws DukeExceptions {
                 commandType = new TaskSelectCommand(taskIndex, projectIndex);
             }
             break;
+        case "description":
+            if (isProjectListView) {
+                System.out.println("Not in Task View!");
```

Comment:
Should not be "Task View"?

Reply:
It is task view iirc. isProjectListView refers to project view so means not in task view